crs_application/resources/crs_process.scxml

```diff
@@ -0,0 +1,195 @@
+            <qt:editorinfo geometry="-38.58;-27.44;-76.20;-50;164.08;100" scenegeometry="938.41;305.63;862.21;255.63;164.08;100"/>
+            <transition type="external" event="sm_done" target="Exec_Media_Change">
+                <qt:editorinfo movePoint="3.70;9.87" endTargetFactors="12.03;51.21"/>
+            </transition>
+        </state>
+    </state>
+    <state id="Part_Rework">
+        <qt:editorinfo geometry="938.22;204.12;-120.73;-190.97;511.03;412.54" scenegeometry="938.22;204.12;817.49;13.15;511.03;412.54"/>
+        <state id="Waits_User_Assesment">
+            <qt:editorinfo geometry="-35.56;0.57;-60;-79.89;169.51;132.94" scenegeometry="902.66;203.67;842.66;123.78;169.51;132.94"/>
+            <transition type="external" event="user_approves" target="PR_Move_Robot">
+                <qt:editorinfo movePoint="39.77;-17.33" localGeometry="20.74;-89.15" endTargetFactors="11.17;23.52"/>
+            </transition>
+        </state>
+        <state id="PR_Move_Robot">
+            <qt:editorinfo geometry="175.01;-75.52;-60;-50;120;100" scenegeometry="1113.23;127.58;1053.23;77.58;120;100"/>
+            <transition type="external" event="sm_done" target="PR_Capture_Image">
+                <qt:editorinfo movePoint="47.92;12.23" localGeometry="150.11;0.38" endTargetFactors="64.55;12"/>
+            </transition>
+        </state>
+        <state id="PR_Capture_Image">
+            <qt:editorinfo geometry="310.30;15.30;-68.34;-50;128.34;100" scenegeometry="1248.52;219.42;1180.18;169.42;128.34;100"/>
+            <transition type="external" event="sm_done" target="PR_Check_Queue">
+                <qt:editorinfo movePoint="-13.26;-22.43" startTargetFactors="10.15;30.49" localGeometry="-91.65;0" endTargetFactors="53.59;11.51"/>
+            </transition>
+        </state>
+        <state id="PR_Check_Queue">
+            <qt:editorinfo geometry="159.03;65.91;-60;-50;120;100" scenegeometry="1097.26;270.03;1037.26;220.03;120;100"/>
+            <transition type="external" event="sm_next" target="PR_Capture_Image">
+                <qt:editorinfo movePoint="50.98;32.63" startTargetFactors="91.89;68.64" endTargetFactors="15.81;73.56"/>
+            </transition>
+            <transition type="external" event="sm_done" target="TrimToolPath">
+                <qt:editorinfo movePoint="35.69;18.35" localGeometry="1.83;81.86" endTargetFactors="9.86;52.34"/>
+            </transition>
+        </state>
+        <state id="PR_Preview">
+            <qt:editorinfo geometry="19.77;130.55;-80;-43.08;130.94;113.08" scenegeometry="957.99;334.67;877.99;291.59;130.94;113.08"/>
+            <transition type="external" event="user_approves" target="Motion_Planning">
+                <qt:editorinfo movePoint="-30.59;99.92" startTargetFactors="9.54;80.42" localGeometry="-59.50;0;-59.50;155.15" endTargetFactors="94.18;3.36"/>
+            </transition>
+            <transition type="external" event="user_rejects" target="Ready">
```

Comment:
I am confused as to what exactly user_rejects means here, does this mean the user rejects doing any more sanding or that the user rejects the tool path. Seems like there should be some sort of outgoing signal from the Part_Rework box that indicates the entire process is finished.

Reply:
I agree, this may not be the correct workflow.  This SM will get more fleshed out as the development of this application progresses.